src/v2/Components/v2/ArtworkFilter/ArtworkFilters/__tests__/MaterialsTermFilter.jest.tsx

```diff
@@ -78,5 +78,37 @@ describe("MaterialsTermFilter", () => {
       expect(context.filters.materialsTerms).toContain("oil")
       expect(context.filters.materialsTerms).toContain("canvas")
     })
+
+    it("autocompletes available options", async () => {
+      const wrapper = getWrapper({
+        aggregations: [
+          {
+            slice: "MATERIALS_TERMS",
+            counts: [
+              {
+                name: "Oil",
+                count: 10,
+                value: "oil",
+              },
+              {
+                name: "Canvas",
+                count: 5,
+                value: "canvas",
+              },
+            ],
+          },
+        ],
+      })
+
+      const input = wrapper.find("input")
+      expect(input).toBeDefined()
+
+      input.simulate("focus")
+      // error: returns 0 nodes 🤔
```

Comment:
Wanted to get a unit test around this, but struggling to get Enzyme to do what I want. I figured a full `mount` would produce the entire component subtree here, but apparently not?

Since this is already tested via the FacetAutosuggest spec, I'm fine ditching this, but figured I'd be a good sport first. Does anything stand out here?

Reply:
There's a min. number of items for which we won't display the autosuggest input box (https://github.com/artsy/force/blob/81baecc2f67518caaca15d0d55c02144399aa995/src/v2/Components/v2/ArtworkFilter/ArtworkFilters/FacetAutosuggest.tsx#L90), so that could be the case here?

But yea, I would think we prob can get away w/o this spec at all in favor of the specs in the autosuggest component, or maybe asserting that the `FacetAutosuggest` component is rendered in this tree or something.

Reply:
Ah yeah, it was totally `MIN_ITEMS`! Easy fix, so I left the test in.